ctapipe/io/datawriter.py

```diff
@@ -53,25 +57,29 @@ def write_reference_metadata_headers(obs_ids, subarray, writer, is_simulation):
         SubarrayDescription to get metadata from
     writer: HDF5TableWriter
         output
+    data_levels: List[DataLevel]
+        list of data levels that were requested/generated
+        (e.g. from `DataWriter.datalevels`)
     """
     activity = PROV.current_activity.provenance
+    category = "Sim" if is_simulation else "Other"
```

Comment:
Are these categories defined somewhere? Might an enum make sense?

Reply:
yes, they are defined in metadata.py - it only allows "Sim", "A","B","C".  Could use the enum explicitly here though

Reply:
eventually they will maybe need to be set by a user option (since it A/B/C refers to which *official* pipeline produced them: realtime, next-day, offline, but for now don't have much meaning, so "other" is the default).